### PR TITLE
feat(jti): listing refresh token

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
[related pr in fence](https://github.com/uc-cdis/fence/pull/61)
Change UserPorfile page to help users manage list of refresh tokens
r @philloooo @frickjack

### CODE REVIEW

File: src/Popup/Popup.jsx [hunk 1]
```diff
@@ -10,9 +10,27 @@ const button = css`
   cursor: pointer;
   margin-left: 1em;
 `;
+
 export const Button = styled.a`
   ${button};
 `;
+
```
Comment (frickjack) [inline src/Popup/Popup.jsx]: LeftButton and LeftCancelButton aren't used - remove?

File: src/Popup/SavePopup.jsx [hunk 2]
```diff
@@ -15,9 +15,18 @@ const SavePopupBox = styled.section`
   overflow: hidden;
 `;
 
+const Cell = styled.li`
```
Comment (frickjack) [inline src/Popup/SavePopup.jsx]: Cell is not used - remove

File: src/UserProfile/UserProfile.jsx [hunk 3]
```diff
@@ -182,60 +206,62 @@ const UserProfile = ({ user, userProfile, popups, submission, onCreateKey,
...
           {NO_ACCESS_MSG}
         </div>
       }
       {
-        userProfile.access_key_pairs !== undefined && userProfile.access_key_pairs !== [] &&
+        userProfile.jtis !== undefined && userProfile.jtis !== [] &&
         <KeyPairTable>
           {
-            popups.key_delete_popup === true &&
+            popups.delete_token_popup === true &&
```
Comment (frickjack) [inline src/UserProfile/UserProfile.jsx]: Swith this to camel case

File: src/UserProfile/UserProfile.jsx [hunk 4]
```diff
@@ -182,60 +206,62 @@ const UserProfile = ({ user, userProfile, popups, submission, onCreateKey,
...
               message={CONFIRM_DELETE_MSG}
               error={jsonToString(userProfile.delete_error)}
-              onConfirm={() => onDeleteKey(userProfile.request_delete_key, popups.keypairsApi)}
+              onConfirm={() => onDeleteKey(userProfile.request_delete_jti,
+                userProfile.request_delete_exp,
+                popups.keypairsApi)}
               onCancel={() => {
                 onClearDeleteSession();
-                onUpdatePopup({ key_delete_popup: false });
+                onUpdatePopup({ delete_token_popup: false });
```
Comment (frickjack) [inline src/UserProfile/UserProfile.jsx]: Use camel case with javascript

File: src/UserProfile/ReduxUserProfile.js [hunk 5]
```diff
@@ -30,48 +30,53 @@ export const fetchAccess = () =>
...
+  exp,
 });
 
 const clearDeleteSession = () => ({
   type: 'CLEAR_DELETE_KEY_SESSION',
 });
 
 
-const deleteKey = (accessKey, keypairsApi) =>
+const deleteKey = (jti, exp, keypairsApi) =>
```
Comment (frickjack) [inline src/UserProfile/ReduxUserProfile.js]: Add a comment explaining what 'jti' and 'exp' are?
@param {string} jti is the token id
@param {number} exp is expiration?  - why is this needed?

File: src/UserProfile/ReduxUserProfile.test.jsx [hunk 6]
```diff
@@ -81,16 +131,16 @@ describe('the userProfile component', () => {
     const expectedActions = [
       {
         type: 'CREATE_SUCCEED',
-        access_key_pair: expectedData,
-        str_access_key_pair: parseKeyToString(expectedData),
+        refresh_cred: expectedData,
```
Comment (frickjack) [inline src/UserProfile/ReduxUserProfile.test.jsx]: camel case

File: src/UserProfile/ReduxUserProfile.js (hunk 5 above)
Comment (thanh-nguyen-dang) [reply src/UserProfile/ReduxUserProfile.js]: `exp` is required by `blacklist` API
